### PR TITLE
New Home Page & Chain Activity

### PR DESCRIPTION
**Pull request by giulianoconti**
## Description #569 #583 

- New Home Page.
- Transaction Activity section added.
- New components added such as download in sankey chart, new select, etc.

newHome

### CODE REVIEW

File: src/components/atoms/MaintenanceMessage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const MaintenanceMessage = () => {
+  return (
+    <div className="maintenance-message">
+      <div className="maintenance-message-icon">
+        <div className="maintenance-message-icon-dot" />
+      </div>
+
+      <p className="maintenance-message-text">
+        due to maintanace the wormhole scanner will be own on may 15th between 15:00 utc - 18:00 utc
```
Comment (valentinoConti) [inline src/components/atoms/MaintenanceMessage/index.tsx:11]: "the wormhole scanner will be own" 🤔

I'm almost sure that text is wrong,  isnt it?

File: src/components/molecules/ChainActivity/index.tsx [hunk 2]
```diff
@@ -0,0 +1,874 @@
...
+  10003: "#405870",
+  10004: "#0052FF",
+  10005: "#FF0420",
+  10006: "#627EEA",
+  10007: "#8247E5",
+};
+
+const DAY_IN_MILLISECONDS = 86400000;
+
+function startOfDayUTC(date: Date) {
```
Comment (valentinoConti) [inline src/components/molecules/ChainActivity/index.tsx:861]: Maybe this and the following 2 functions should be in `utils/date.ts` to have every date functions there for other things

**Review by valentinoConti — APPROVED** [review]
🔥🔥🔥🔥🔥🔥

File: src/components/atoms/MaintenanceMessage/index.tsx (hunk 1 above)
Comment (giulianoconti) [reply src/components/atoms/MaintenanceMessage/index.tsx:11]: In the figma it is like this, but it is just an example, when we have to place an alert, this text will change

**Pull request merged**